src/nxt_conf_validation.c

```diff
@@ -2913,13 +2917,11 @@ nxt_conf_vldt_object_iterator(nxt_conf_validation_t *vldt,
 
     for ( ;; ) {
         member = nxt_conf_next_object_member(value, &name, &index);
-
         if (member == NULL) {
             return NXT_OK;
         }
 
         ret = validator(vldt, &name, member);
-
         if (ret != NXT_OK) {
             return ret;
         }
```

Comment:
And this hunk should probably go also, it's harmless but it's also not really related now...

Reply:
I noticed that one, and I liked it.  It's a small reminder of where the validator is called, and where this could have been fixed instead.  We may need that in the future if other functions start needing writable strings.  I'll keep this hunk.